docs/reference/migration/migrate_7_0/java_time.asciidoc

```diff
@@ -0,0 +1,47 @@
+[float]
+[[breaking_70_java_time_changes]]
+=== Replacing joda time with java time
+
+Since Java 8 there is a dedicated `java.time` package, which is superior to
+the joda time library, that has been used so far in Elasticsearch. One of
```

Comment:
Nit: joda time -> Joda-Time (according to https://www.joda.org/joda-time/) - I won't point this out for the rest of the doc.